packages/chat-client/src/controllers/engine.ts

```diff
@@ -371,6 +374,39 @@ export class ChatEngine extends IChatEngine {
     return inviteId;
   };
 
+  public unregisterIdentity: IChatEngine["unregisterIdentity"] = async ({
+    account,
+  }) => {
+    const iat = Date.now();
+    const keys = this.client.chatKeys.get(account);
+    const didPublicKey = composeDidPkh(account);
+    const unregisterIdentityPayload: InviteKeyClaims = {
+      iat,
+      exp: jwtExp(iat),
+      iss: encodeEd25519Key(keys.identityKeyPub),
+      aud: this.keyserverUrl,
+      pkh: didPublicKey,
+      act: "unregister_identity",
+    };
+
+    const idAuth = await this.generateIdAuth(
+      account,
+      unregisterIdentityPayload
+    );
+
+    const url = `${this.keyserverUrl}/identity`;
+
+    const response = await axios.delete(url, {
```

Comment:
Are we using axios to make sure the code runs both client and server side ?

Reply:
Yes, and it's the standard around our repos.